docker/data-science/notebooks/ch08/1.signals-boosting.ipynb

```diff
@@ -177,23 +167,34 @@
       "Status: Success\n",
       "Signals Aggregation Completed!\n"
      ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<Response [415]>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
     "def aggregate_signals(signals_collection, collection_name, signals_agg_query):\n",
     "    print(\"Aggregating Signals to Create Signals Boosts...\")\n",
     "    create_view(signals_collection, \"signals\")\n",
     "    collection = engine.create_collection(collection_name)\n",
-    "    collection.write_from_sql(signals_agg_query)\n",
+    "    collection.write(from_sql(signals_agg_query))\n",
+    "    time.sleep(5)\n",
```

Comment:
Is this temporary for the final solution (versus calling commit with a "waitSearcher=true" parameter to block until the docs are written and available).

I'd suggest attempting the waitSearcher option versus and using time.sleep as a fallback it that doesn't work.

Reply:
Also, whatever the solution, we (probably?) want to move it inside the collection.write(...) method so whatever level of flushing/waiting can be implemented per-engine. Som engines probably have ACID guarantees and this is a non-issue, and some may just need a time.sleep(LONG_TIME) implementation.

Reply:
Adding a waitSearcher=true to the commit function